.circleci/config.yml

```diff
@@ -258,6 +258,46 @@ jobs:
               run --rm --no-deps app \
                 black src/richie/apps src/richie/plugins sandbox --check
 
+  extract-back-strings:
+    machine: true
+    working_directory: ~/fun
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - docker-debian-images-dev-{{ .Revision }}
+      - run:
+          name: Load images to docker engine
+          command: |
+            docker load < docker/images/dev/richie.tar
+      - run:
+          name: Generate a POT file from strings extracted from the project
+          command: make messages
```

Comment:
:warning: Using this `Makefile`'s rule, you'll use the development's `docker-compose` configuration. This is not what you want. Instead you can run the `makemessages` management command using the current docker image:

```
docker run --rm \
    --user="$(id -u)" \
    --volume "$PWD:/app" \
    --workdir /app/src/richie \
    "richie:${CIRCLE_SHA1}-dev" \
        python /app/sandbox/manage.py makemessages --keep-pot
```